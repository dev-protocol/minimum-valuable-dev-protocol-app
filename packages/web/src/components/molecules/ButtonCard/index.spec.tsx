import React from 'react'
import { render } from '@testing-library/react'
import { ButtonCard } from '.'
import 'src/__mocks__/window/matchMedia.mock'

describe(`${ButtonCard.name}`, () => {
  test('Snapshot', () => {
    const component = render(<ButtonCard label="Cancel Staking" onClick={() => {}} buttonLabel="Cancel" />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
