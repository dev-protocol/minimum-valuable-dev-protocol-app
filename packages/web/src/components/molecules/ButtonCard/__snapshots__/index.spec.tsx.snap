// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonCard Snapshot 1`] = `
<body>
  <div>
    <div
      class="ant-card ant-card-bordered"
    >
      <div
        class="ant-card-body"
      >
        <div
          class="ant-row"
        >
          <div
            class="ant-col"
            style="display: flex; align-items: center; padding: 16px 0px; flex: 1 1 252px;"
          >
            <div
              style="font-size: 24px; line-height: 32px; color: rgb(0, 0, 0);"
            >
              Cancel Staking
            </div>
          </div>
          <div
            class="ant-col"
            style="padding: 16px 0px; flex: 1 1 252px;"
          >
            <button
              class="ant-btn ant-btn-primary ant-btn-lg"
              type="button"
            >
              <span>
                Cancel
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
